integrated_channels/sap_success_factors/migrations/0002_auto_20170222_1102.py

```diff
@@ -0,0 +1,64 @@
+# -*- coding: utf-8 -*-
```

Comment:
One thing we can do before the initial merge of a new app is merge the migration files into a single 0001_initial.py.  This basically involves deleting all of the existing migration files and running makemigrations one last time to pick up all of the changes.